src/revlanguage/distributions/phylogenetics/character/Dist_CTMC.h

```diff
@@ -0,0 +1,72 @@
+#ifndef Dist_CTMC_H
```

Comment:
Definitely add a help page for this. It's not clear to me how this is different from PhyloCTMC.

Reply:
It's for a single population/branch, so no tree is used. I'm working on something to use this process to model demographic processes. I've added a bit of description in commit `47d45a38`.